prismic-model/js/parts/body.js

```diff
@@ -29,11 +29,17 @@ const featuredLabel = {
   display: 'Featured',
 };
 
+const showClosingTimesLabel = {
+  name: 'showClosingTimes',
+  display: 'Show closing times',
```

Comment:
why not `showClosingTimes`?

Reply:
[See the note on the modelling above](https://github.com/wellcometrust/wellcomecollection.org/pull/4405/files#r277585701)